app/models/config/__init__.py

```diff
@@ -26,6 +26,7 @@ class Config:
     time_to_cancel_actions: int = 60
     time_to_remove_temp_messages: int = 30
     report_karma_award: int = 0
+    time_to_remove_report_karma_award_messages: int = 300  # If time in seconds less than 1, then messages will not be deleted
```

Comment:
```suggestion
    report_award_cleanup_delay: int = 3600  # If time in seconds less than 1, then messages will not be deleted
```

Reply:
@bomzheg, what do you think about this suggestion?

There was also a suggestion to use `reward_message_ttl`.